instant-xml/tests/all.rs

```diff
@@ -4,10 +4,12 @@ use instant_xml::{FromXml, ToXml};
 struct Unit;
 
 #[derive(Debug, Eq, PartialEq, ToXml)]
-#[xml(namespace("URI"))]
+#[xml(namespace("URI", bar = "BAZ", foo = "BAR"))]
 struct StructWithNamedFields {
     flag: bool,
+    #[xml(namespace("bar"))]
```

Comment:
I think you meant `xml(namespace(bar))` here, in order to reference the previously defined `BAZ` namespace through a prefix? In that case the resulting serialization should include `xmlns:bar="BAZ" in the `StructWithNamedFields` start tag, and yield `<bar:string>test</bar:string>` here.